cmake/ConfigureChecks.cmake

```diff
@@ -994,6 +1000,86 @@ python_platform_test(
   DIRECT
   )
 
+
+# Check whether assembler supports .cfi_* directives
+set(check_src ${PROJECT_BINARY_DIR}/CMakeFiles/have_as_cfi_pseudo_op.c)
+file(WRITE ${check_src} "int main() {
+    __asm__ __volatile__ (\".cfi_startproc\n\t.cfi_endproc\");
+}
+")
+python_platform_test(
+  HAVE_AS_CFI_PSEUDO_OP
+  "Checking whether assembler supports .cfi_* directives"
+  ${check_src}
+  DIRECT
+  )
+
+# Check whether assembler supports PC relative relocs
```

Comment:
If at some point we either create `libffi-cmake-buildsystem` or contribute our changes upstream.

A small nits would be update comment like:

```
# Check whether assembler supports PC relative relocs
```
into 

```
# libffi specific: Check whether assembler supports PC relative relocs
```